src/components/Faq/Faq.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import uuidv4 from 'uuid/v4'
+import { TitleLarge, TitleMedium, Body } from '../index'
+
+import styles from './Faq.module.scss'
+
+export const Faq = ({ questions, open }) => {
+  return (
+    <div className={styles.root}>
+      <div className={styles.leftColumn}>
+        <TitleMedium.Serif.Book500>FAQs</TitleMedium.Serif.Book500>
+      </div>
+
+      <div className={styles.rightColumn}>
+        <div className={styles.phoneTitle}>
+          <TitleLarge.Serif.Book500>FAQs</TitleLarge.Serif.Book500>
+        </div>
+        {questions.map((q, index) => {
+          const startOpen = open && index === 0
+          return (
+            <Details summary={q.summary} open={startOpen} key={uuidv4()}>
+              {q.content}
+            </Details>
+          )
+        })}
+      </div>
+    </div>
+  )
+}
+
+Faq.defaultProps = {
+  questions: [],
+  open: true,
+}
+
+Faq.propTypes = {
+  questions: PropTypes.arrayOf(
+    PropTypes.shape({
+      summary: PropTypes.string,
+      content: PropTypes.oneOfType([PropTypes.object, PropTypes.string]),
+    })
+  ),
+  open: PropTypes.bool,
+}
+
+const Details = ({ children, summary, open }) => {
+  const carrotSvg = (
+    <svg viewBox="0 0 19 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Non Blocking -- I've created svg-sprite-loader which means you can now put this in `src/svgs` and then use sprite loader to pull it in (see LogoInline aka Logo component).

That said, I don't know that doing this should block this contribution.